### ISSUE TITLE
Increased speaking rate when reading stressed words

### ISSUE THREAD

**Issue by beqabeqa473** [poretsky/ru_tts#16]
Hello.

In current version of ru_tts, we are seeing an increased speaking rate when turning on stress dictionary support and when reading stressed words.
Would it be possible to increase length of a stressed wowel or make it configurable, however personally i don't see a point of doing that.

Thanks.

**Comment by poretsky** [comment]
In fact, I don't treat it as an issue.

Technically, control of the stressed vowels length is not impossible,
but I do not plan to implement it, since like you I see no point in such
facility. At least, I think, it isn't worth efforts.

**Comment by beqabeqa473** [comment]
I don't see a point to do it controlable, but if you don't want to
implement it, it is also understandable. I could implement it myself
if you would be able to give me a hint where to look. Because, some
people in feedback was writing about this problem, that if they have
set speed at maximum where there are not clicks, turning on stressing
support makes wowels faster and clicks are appearing.

Thanks.

On 9/30/23, Igor B. Poretsky ***@***.***> wrote:
> In fact, I don't treat it as an issue.
>
> Technically, control of the stressed vowels length is not impossible,
> but I do not plan to implement it, since like you I see no point in such
> facility. At least, I think, it isn't worth efforts.
>
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/poretsky/ru_tts/issues/16#issuecomment-1741834491
> You are receiving this because you authored the thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by poretsky** [comment]
Thus, they should decrease speed when using dictionary. What the
problem?

Concerning a hint where to look, I'd suggest module
speechrate_control.c. Its name itself serves as a suggestion. Or, maybe
something in intonator.c can also relate to the subject. But it may
turn out not as simple as wanted. Doubt if it worth efforts at all.

**Comment by beqabeqa473** [comment]
No, the overal problem is that in this case rhytmics is somehow
becomes unusual. i mean if we listen ru_tts without stresses it is
speaking as usual, but if we turn stresses on, it becomes noticeable
faster and in this case speed is variable, for example if word is not
in rulex it slows down and if stress is found it again speeds up.
That is my questions if it is possible to slightly lengthen this
stressed wowel to avoid making synth to speak faster than usual, at
least to be not so noticeable.

Thanks.

On 10/1/23, Igor B. Poretsky ***@***.***> wrote:
> Thus, they should decrease speed when using dictionary. What the
> problem?
>
> Concerning a hint where to look, I'd suggest module
> speechrate_control.c. Its name itself serves as a suggestion. Or, maybe
> something in intonator.c can also relate to the subject. But it may
> turn out not as simple as wanted. Doubt if it worth efforts at all.
>
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/poretsky/ru_tts/issues/16#issuecomment-1741929713
> You are receiving this because you authored the thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili
